launch_testing_ros/launch_testing_ros/wait_for_topics.py

```diff
@@ -63,9 +66,13 @@ def __init__(self, topic_tuples, timeout=5.0):
         self.__ros_spin_thread.start()
 
     def _prepare_ros_node(self):
-        node_name = '_test_node_' +\
-            ''.join(random.choices(string.ascii_uppercase + string.digits, k=10))
-        self.__ros_node = _WaitForTopicsNode(name=node_name, node_context=self.__ros_context)
+        node_name = '_test_node_' + ''.join(
+            random.choices(string.ascii_uppercase + string.digits, k=10)
+        )
+        self.__ros_node = _WaitForTopicsNode(
+            name=node_name, node_context=self.__ros_context,
+            max_number_of_messages=self.messages_received_buffer_length
+        )
```

Comment:
nitpick: please put each parameter on a new line when you do something like this, it's really easy to overlook the second parameter in this case.

```suggestion
        self.__ros_node = _WaitForTopicsNode(
            name=node_name,
            node_context=self.__ros_context,
            max_number_of_messages=self.messages_received_buffer_length,
        )
```

And since this is Python, either have a trailing `,` (as I did above) or "cuddle" the closing parenthesis:

```suggestion
        self.__ros_node = _WaitForTopicsNode(
            name=node_name,
            node_context=self.__ros_context,
            max_number_of_messages=self.messages_received_buffer_length)
```